D/DecompUtil/build_tarballs.jl

```diff
@@ -0,0 +1,50 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "DecompUtil"
+version = v"0.1.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/dev10110/DecompUtil_C.git", "982449a84f620488fe9d954eb9a6d301dddfff3f")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+apk add eigen-dev
```

Comment:
You should use `Eigen_jll` as a [`BuildDependency`](https://docs.binarybuilder.org/stable/reference/#BinaryBuilderBase.BuildDependency) instead.  See https://docs.binarybuilder.org/stable/build_tips/#Dependencies-for-the-target-system-vs-host-system.

Reply:
updated.